cumulus/polkadot-omni-node/lib/src/cli.rs

```diff
@@ -137,12 +141,17 @@ pub struct Cli<Config: CliConfig> {
 	#[arg(long)]
 	pub dev_block_time: Option<u64>,
 
-	/// EXPERIMENTAL: Use slot-based collator which can handle elastic scaling.
+	/// DEPRECATED: This feature has been stabilized, pLease use `--authoring slot-based` instead.
 	///
+	/// Use slot-based collator which can handle elastic scaling.
 	/// Use with care, this flag is unstable and subject to change.
 	#[arg(long)]
 	pub experimental_use_slot_based: bool,
 
+	/// Authoring style to use.
+	#[arg(long, default_value_t = AuthoringStyle::Lookahead)]
```

Comment:
Why then not make the slot based the default?

Reply:
Lookahead still builds on forks, I don't want to touch the normal async-backing chains here. This here is for stabilizing the elastic-scaling use-case, I don't want to mess with too many things at once.

IMO we can make slot-based the default in the release after next.

Reply:
I see no reason to hold this from being the default, unless we haven't tested it enough and we are concerned that async backing chains will suffer. We should run this on all collators that Parity runs on async backing chains to get some data, otherwise it seems we will have the same concern next time.